docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Text/TokenizeIntoCharacters.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class TokenizeIntoCharacters
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Create an empty data sample list. The 'TokenizeIntoCharactersAsKeys' does not require training data as
+            // the estimator ('TokenizingByCharactersEstimator') created by 'TokenizeIntoCharactersAsKeys' API is not a trainable estimator.
+            // The empty list is only needed to pass input schema to the pipeline.
+            var samples = new List<TextData>();
+
+            // Convert sample list to an empty IDataView.
+            var dataview = mlContext.Data.LoadFromEnumerable(samples);
+
+            // A pipeline for converting text into vector of characters.
+            // The 'TokenizeIntoCharactersAsKeys' produces result as key type.
+            // 'MapKeyToValue' is need to map keys back to their original values.
+            var textPipeline = mlContext.Transforms.Text.TokenizeIntoCharactersAsKeys("CharTokens", "Text", useMarkerCharacters: false)
+                .Append(mlContext.Transforms.Conversion.MapKeyToValue("CharTokens"));
+
+            // Fit to data.
+            var textTransformer = textPipeline.Fit(dataview);
+
+            // Create the prediction engine to get the character vector from the input text/string.
+            var predictionEngine = mlContext.Model.CreatePredictionEngine<TextData, TransformedTextData>(textTransformer);
+
+            // Call the prediction API to convert the text into characters.
+            var data = new TextData() { Text = "ML.NET's TokenizeIntoCharactersAsKeys API splits text/string into characters." };
+            var prediction = predictionEngine.Predict(data);
+
+            // Print the length of the character vector.
+            Console.WriteLine($"Number of tokens: {prediction.CharTokens.Length}");
+
+            // Print the character vector.
+            Console.WriteLine($"\nCharacter Tokens: {string.Join(",", prediction.CharTokens)}");
+
+            //  Expected output:
+            //   Number of tokens: 112
+            //   Character Tokens: M,L,.,N,E,T,',s,<?>,T,o,k,e,n,i,z,e,I,n,t,o,C,h,a,r,a,c,t,e,r,s,A,s,K,e,y,s,<?>,A,P,I,<?>,
+            //                     s,p,l,i,t,s,<?>,t,e,x,t,/,s,t,r,i,n,g,<?>,i,n,t,o,<?>,c,h,a,r,a,c,t,e,r,s,.
```

Comment:
><?> [](start = 73, length = 3)

do we really present `space` as `<?>` ?

Reply:
Its a unit separator special character.
https://github.com/dotnet/machinelearning/blob/366332047340bdb12cf5fe2a11f9a6e257af9b84/src/Microsoft.ML.Transforms/Text/TokenizingByCharacters.cs#L90

Reply:
sorry! this is the control character used instead of spaces. Please disregard my previous comments 
https://github.com/dotnet/machinelearning/blob/366332047340bdb12cf5fe2a11f9a6e257af9b84/src/Microsoft.ML.Transforms/Text/TokenizingByCharacters.cs#L275

Reply:
would be nice to left some comment about that.

---
In reply to: [270190306](https://github.com/dotnet/machinelearning/pull/3123#discussion_r270190306) [](ancestors = 270190306)